### PR TITLE
chore(demo): add external stackblitz playground

### PR DESCRIPTION
**Pull request by splincode**
Close https://github.com/taiga-family/taiga-ui/issues/9083

### CODE REVIEW

File: projects/demo/src/modules/app/stackblitz/starter/index.ts [hunk 1]
```diff
@@ -1,56 +1,16 @@
...
-            {
-                newWindow: false,
-                openFile: appPrefix`app.template.html`,
-                hideExplorer: true,
-            },
-        );
-    }
+export default class Page {
+    protected readonly redirect = inject(WA_WINDOW).location.replace(
+        'https://stackblitz.com/~/github.com/taiga-family/taiga-playground?file=src/app/app.component.html',
```
Comment (waterplea) [inline projects/demo/src/modules/app/stackblitz/starter/index.ts:14]: I think we only need it when somebody enters taiga-ui.dev/stackblitz manually, in header we can just have a regular target="_blank" link to this address, right?

**Review by waterplea — APPROVED** [review]

**Comment by waterplea** [comment]
Does this PR address this? https://github.com/taiga-family/taiga-ui/issues/9083

File: projects/demo/src/modules/app/stackblitz/starter/index.ts (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/app/stackblitz/starter/index.ts:14]: right

Comment (splincode) [reply projects/demo/src/modules/app/stackblitz/starter/index.ts]: fixed

File: projects/demo/src/modules/app/app.template.html [hunk 2]
```diff
@@ -47,10 +47,11 @@
             <a
                 appearance="icon"
                 iconStart="assets/icons/stackblitz.svg"
+                target="_blank"
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/demo/src/modules/app/app.template.html:51]: fixed

File: projects/demo/src/modules/app/stackblitz/starter/index.ts [hunk 3]
```diff
@@ -1,56 +1,17 @@
...
-                        "@import '@taiga-ui/core/styles/taiga-ui-local.less';",
-                },
-            },
-            {
-                newWindow: false,
-                openFile: appPrefix`app.template.html`,
-                hideExplorer: true,
-            },
-        );
+export default class Page {
```
Comment (nsbarsukov) [inline projects/demo/src/modules/app/stackblitz/starter/index.ts:13]: Do we actually need this page for the new approach ?

Could we just update these lines
https://github.com/taiga-family/taiga-ui/blob/bfe18a1b6776df638037bf19cc8230879de84268/projects/demo/src/modules/app/pages.ts#L52-L58

to the following way ?

```ts
{
    section: 'Documentation',
    title: 'StackBlitz',
    keywords: 'reproduce, issue, bug, sandbox, playground, test',
    route: environment.playground,
    target: '_blank',
},
```

Comment (splincode) [reply projects/demo/src/modules/app/stackblitz/starter/index.ts:13]: `Do we actually need this page for the new approach ?` @waterplea fyi

Comment (waterplea) [reply projects/demo/src/modules/app/stackblitz/starter/index.ts:13]: I believe it's nice to have `https://taiga-ui.dev/stackblitz` as an alias to that playground. We definitely shared it here and there, let's keep it working.

File: projects/demo/src/modules/app/app.template.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (mdlufy) [reply projects/demo/src/modules/app/app.template.html:51]: Not enough only `noreferrer`? Lighthouse not satisfy spec?

https://html.spec.whatwg.org/multipage/links.html#link-type-noreferrer
`It indicates that no referrer information is to be leaked when following the link and also implies the noopener keyword behavior under the same conditions.`

[the remark this answers is not in the document]
Comment (splincode) [reply projects/demo/src/modules/app/app.template.html:51]: fixed

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**